index.md

```diff
@@ -1,51 +1,70 @@
-# Health Record Ingestion service
+# Introduction
 
-The HRI is a deployment ready service for streaming Health-related data into the cloud. It provides a "front door" for "Data Integrators" to send data into the IBM Cloud, while supporting both batch-processing and data streaming workflows. It provides features to initiate and track the movement of a dataset into the IBM cloud for both ["Data Integrators"](glossary.md#data-integrator) and ["Data Consumers"](glossary.md#data-consumer). 
+The **[Health Record Ingestion service](glossary.md#health-record-ingestion-hri)** is a deployment-ready service for streaming healthcare-related data into the cloud. It provides a "front door" for [Data Integrators](glossary.md#data-integrator) to send data into the IBM&reg; Cloud, while supporting both batch processing and data streaming workflows. It provides features to initiate and track the movement of a dataset into the IBM Cloud for both Data Integrators and [Data Consumers](glossary.md#data-consumer). 
 
-The key features are:
-- Streaming - all data is streamed
-- Batch support - a collection of health data records can be streamed and processed together
-- Multitenancy - supports segregation of data by tenant and Data Integrator
+## Key features
+
+- **Streaming:** All data is streamed.
+- **Batch support:** A collection of health data records can be streamed and processed together.
+- **Multi-tenancy:** Supports segregation of data by [tenant](glossary.md#tenant) and Data Integrator
+
+## Key technologies
+
+- [**Event Streams**](glossary.md#event-streams): This is an IBM Cloud-based [Apache Kafka](https://kafka.apache.org/) managed service, the technology used for producing and consuming the data streams.
+- [**IBM Cloud Functions**](glossary.md#ibm-cloud-functions): Using these, the Health Record Ingestion service exposes a serverless RESTful Management API that is used to control and configure the system. 
+- [**Elasticsearch**](glossary.md#elasticsearch): This is the distributed NoSQL data store that is used to store information about batches.
+
+## IBM Cloud dependencies
+
+The Health Record Ingestion service was developed on the IBM Cloud and currently does not support running on other public or private clouds. However, as part of Project Alvearie, the Health Record Ingestion service is being transitioned into an open source project, with a goal of supporting public and private cloud deployments. The latest version has been pushed to GitHub. The team continues to work on moving all development activities into the open, and making the Health Record Ingestion service support other clouds. For more information about future plans, see the [Roadmap](roadmap.md).   
+
+## Core architecture
+
+
+**Figure: Components of the Health Record Ingestion service**
```

Comment:
Can we change this to 'Core Architecture' and move it after the diagram?  I'd like to move all the 'Figure' labels to after the diagrams.